src/migration/1717589112159-SourceStatsTrigger.ts

```diff
@@ -0,0 +1,128 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SourceStatsTrigger1717589112159 implements MigrationInterface {
+  name = 'SourceStatsTrigger1717589112159';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(`
+        CREATE OR REPLACE FUNCTION increment_squad_posts_count()
```

Comment:
Not sure if maybe we should delete and re-add with better naming?
`increment_source_posts_count` (but only if it doesn't add much overhead)